### PR TITLE
Publish re-ad messages on the additions feed

### PR DESCRIPTION
**Pull request by Powersource**
for https://github.com/ssbc/ssb-tribes2/issues/79 Hopefully fixes it, but we need to verify in #78 

tests were passing before the first commit

### CODE REVIEW

**Comment by Powersource** [comment]
@arj03 no it's re-add, no read :P

File: index.js [hunk 1]
```diff
@@ -177,91 +177,70 @@ module.exports = {
...
-        get(groupId, (err, { writeKey: oldWriteKey } = {}) => {
+        const excludeContent = {
+          type: 'group/exclude',
+          excludes: feedIds,
+          recps: [groupId],
+        }
+        const excludeOpts = {
+          tangles: ['members'],
+          isValid: isExclude,
+        }
```
Comment (mixmix) [inline index.js:188]: I think we need to think about where we want the exclude published... (which feed).
I have a feeling it should be in the same feed as the additions... but I can't quite justify.
I think it's that we want to be sure we have all the messages for each epoch membership stuff and sprinkling the around might lead to funny edge cases?

what do you think?

File: test/exclude-members.test.js [hunk 2]
```diff
@@ -160,6 +145,32 @@ test('add and remove a person, post on the new feed', async (t) => {
 
   t.equal(post.text, 'post', 'found post on second feed')
 
+  const aliceAdditions = await p(alice.metafeeds.findOrCreate)({
+    purpose: 'group/additions',
+  })
+  const msgsFromAdditions = await alice.db.query(
+    where(author(aliceAdditions.id)),
+    toPromise()
+  )
```
Comment (mixmix) [inline test/exclude-members.test.js:154]: :pineapple: it's cool seeing different ways of querying around the space + testing. I would have got a different way but this is tidy (looking within a particular sub-feed).
db.query is so sweet
